src/tests/cmocka/test_sdap_initgr.c

```diff
@@ -195,7 +195,15 @@ struct sdap_get_initgr_state *prepare_state(struct test_sdap_initgr_ctx *ctx,
     }
     assert_non_null(state->dom);
     assert_non_null(recent_dom_info);
-    recent_dom_info->next = NULL;
```

Comment:
I could not see such warning in #371 and in was not part of previous patchset.
You might hide some bugs in test. Therefore NACK to this patch.

Rest of patch set is good and can be pushed

Reply:
Okay, this patch has been dropped from this PR.